src/tests/components/ForecastDetails.test.js

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import { render } from "@testing-library/react";
+import ForecastDetails from "../../components/ForecastDetails";
+
+describe("ForecastDetails", () => {
+  const validProps = {
+    date: 111111,
+    temperature: {
+      min: 12,
+      max: 22
+    },
+    humidity: 28,
+    wind: {
+      speed: 12,
+      direction: "south"
+    }
+  };
+
+  it("renders correctly", () => {
+    const { asFragment } = render(<ForecastDetails forecast={validProps} />);
+
+    expect(asFragment()).toMatchSnapshot();
+  });
+
+  it("renders correct values for props", () => {
+    const { getByText } = render(<ForecastDetails forecast={validProps} />);
+
+    expect(getByText("Thu Jan 01 1970")).toHaveClass("forecast-details__date");
```

Comment:
This line is failing because you have the text wrapped in a h2 element, so the classname does not directly apply. You could fix it by changing the assertion:
```
expect(getByText("Thu Jan 01 1970")).toBeInstanceOf(HTMLHeadingElement);
```